### PR TITLE
Update to sonar.version 6.2

### PR DESCRIPTION
**Pull request by janos-ss**
Just the minimum changes to compile and pass tests.

### CODE REVIEW

File: sonar-javascript-plugin/src/test/java/org/sonar/plugins/javascript/lcov/CoverageSensorTest.java [hunk 1]
```diff
@@ -179,46 +173,38 @@ public void test_unresolved_path() {
...
   public void test_no_report_path_no_force_zero() {
     context.setSettings(new MapSettings());
     utCoverageSensor.execute(context, linesOfCode, RUN_WITH_SQ_6_1);
-    assertThat(context.lineHits("moduleKey:file1.js", CoverageType.UNIT, 1)).isNull();
+    assertThat(context.lineHits("moduleKey:file1.js", 1)).isNull();
 
     context.setSettings(new MapSettings().setProperty(JavaScriptPlugin.LCOV_UT_REPORT_PATH, UT_LCOV));
     itCoverageSensor.execute(context, linesOfCode, RUN_WITH_SQ_6_1);
-    assertThat(context.lineHits("moduleKey:file1.js", CoverageType.UNIT, 1)).isNull();
-    overallCoverageSensor.execute(context, linesOfCode, RUN_WITH_SQ_6_1);
```
Comment (pynicolas) [inline sonar-javascript-plugin/src/test/java/org/sonar/plugins/javascript/lcov/CoverageSensorTest.java]: @janos-ss Why was this line removed?

File: sonar-javascript-plugin/src/test/java/org/sonar/plugins/javascript/JavaScriptSquidSensorTest.java [hunk 2]
```diff
@@ -406,9 +404,8 @@ public void sq_greater_6_1_still_honor_it_ut_coverage_reports() throws Exception
     inputFile("file1.js");
     createSensor().execute(context);
 
-    // note that these coverages will be aggregated on platform side.
-    assertThat(context.lineHits("moduleKey:file1.js", CoverageType.UNIT, 1)).isEqualTo(2);
-    assertThat(context.lineHits("moduleKey:file1.js", CoverageType.IT, 1)).isEqualTo(1);
+    // note: as of SQ 6.2, there is no more distinction of coverage types
+    assertThat(context.lineHits("moduleKey:file1.js", 1)).isEqualTo(6);
```
Comment (janos-ss) [inline sonar-javascript-plugin/src/test/java/org/sonar/plugins/javascript/JavaScriptSquidSensorTest.java]: @pynicolas `(2 + 1) * 2`, where the `* 2` is caused by the `OverallCoverageSensor`. How about we delete that class?

Comment (vilchik-elena) [inline sonar-javascript-plugin/src/test/java/org/sonar/plugins/javascript/JavaScriptSquidSensorTest.java]: @janos-ss Actually that's a bug in our plugin, it should be `3`, but I tend to think it's not critical at all

File: sonar-javascript-plugin/src/test/java/org/sonar/plugins/javascript/lcov/CoverageSensorTest.java [hunk 3]
```diff
@@ -96,46 +96,8 @@ public void report_not_found() throws Exception {
     assertThat(context.lineHits("moduleKey:file1.js", 1)).isNull();
   }
 
-  // note: as of SQ 6.2, there is no more distinction of coverage types
   @Test
-  public void test_ut_coverage() {
```
Comment (pynicolas) [inline sonar-javascript-plugin/src/test/java/org/sonar/plugins/javascript/lcov/CoverageSensorTest.java]: @janos-ss I think these tests are still relevant. I would keep them.

File: sonar-javascript-plugin/src/test/java/org/sonar/plugins/javascript/JavaScriptSquidSensorTest.java (hunk 2 above)
Comment (pynicolas) [inline sonar-javascript-plugin/src/test/java/org/sonar/plugins/javascript/JavaScriptSquidSensorTest.java]: @janos-ss I see. Let's keep it like that. We may change that behavior in the future.

File: sonar-javascript-plugin/src/test/java/org/sonar/plugins/javascript/lcov/CoverageSensorTest.java (hunk 3 above)
Comment (janos-ss) [inline sonar-javascript-plugin/src/test/java/org/sonar/plugins/javascript/lcov/CoverageSensorTest.java]: @pynicolas yeah I concluded that too, added them back now

File: sonar-javascript-plugin/src/test/java/org/sonar/plugins/javascript/lcov/CoverageSensorTest.java (hunk 1 above)
Comment (janos-ss) [inline sonar-javascript-plugin/src/test/java/org/sonar/plugins/javascript/lcov/CoverageSensorTest.java]: @pynicolas Partly in anticipation of removing the `OverallCoverageSensor`, and partly because it made the next assertion about line hits in UTs confusing, due to the doubled counting we've seen in another test case too.

I improved that now, by changing the order of executions and assertions of coverage sensors, to this order: IT, UT, overall, instead of IT, overall, UT.

**Comment by pynicolas** [comment]
Thanks @janos-ss !
